datasets/wiki_qa_ar/README.md

```diff
@@ -0,0 +1,141 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- ar
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- question-answering
+task_ids:
+- open-domain-qa
+---
+
+# Dataset Card for WikiQAar
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Discussion of Social Impact and Biases](#discussion-of-social-impact-and-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [WikiQaAr](https://github.com/qcri/WikiQAar)
+- **Repository:** [WikiQaAr](https://github.com/qcri/WikiQAar)
+- **Paper:** 
+- **Point of Contact:** [Ines Abbes
+](abbes.ines@yahoo.com)
+
+### Dataset Summary
+
+Arabic Version of WikiQA by automatic automatic machine translators 
+and crowdsourced the selection of the best one to be incorporated into the corpus
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset is based on Arabic.
+
+## Dataset Structure
+
+### Data Instances
+
+Each data point contains the question and whether the answer is a valid or not.  
+
+### Data Fields
+
+[More Information Needed]
```

Comment:
Could you detail a bit the fields ? Especially the `label` one since the name is not explicit

Reply:
Fixed, let me now if that's ok.